test/src/ui/js/test_Game.js

```diff
@@ -1136,19 +1136,33 @@ test("test_Game.pageAddSkillListFooter", function(assert) {
   });
 });
 
-test("test_Game.pageAddLogFooter", function(assert) {
+test("test_Game.pageAddSkillListFooter", function(assert) {
```

Comment:
Huh. I'm pretty sure that my intent was to add the test, but I must have forgotten to duplicate the original pageAddLogFooter test. My bad. I'll add that back in.